tests/rpc_test.go

```diff
@@ -67,29 +88,40 @@ func call(t *testing.T, method string, params interface{}) (*Response, error) {
 		return nil, err
 	}
 
-	/* #nosec */
-	res, err := http.Post(addr, "application/json", bytes.NewBuffer(req))
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	decoder := json.NewDecoder(res.Body)
 	var rpcRes *Response
-	err = decoder.Decode(&rpcRes)
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	if rpcRes.Error != nil {
-		return nil, errors.New(rpcRes.Error.Message)
-	}
-
-	err = res.Body.Close()
-	if err != nil {
-		t.Fatal(err)
+	for i := 0; i < 10; i++ {
```

Comment:
what random failures are we trying to avoid here?  if we try to post a transaction with invalid params or something, then it will cause this not to return for 10 seconds

Reply:
hmm, when I run it gets me some random failures, some tests just fail and others dont, when I run again it works....
actually doing the retry did not really fix the issue, so I am checking what can be the real root of the problem

Reply:
so I removed all the timeout and raised #263 to further investigate and fix this errs.